.editorconfig

```diff
@@ -167,6 +167,9 @@ csharp_space_between_method_declaration_parameter_list_parentheses = false
 csharp_space_between_parentheses = false
 csharp_space_between_square_brackets = false
 
+# CS0618: Type or member is obsolete
+dotnet_diagnostic.CS0618.severity = none
```

Comment:
Hmm is this required? Which obsolete member is being accessed? I am wondering if we can instead just add pragmas at specific violation sites.

Reply:
I can do that instead. Some test libary members were depricated